_posts/2000-01-01-intro.md

```diff
@@ -22,23 +22,23 @@ style: center
 
 See the [release notes page](https://github.com/mockito/mockito/blob/master/doc/release-notes/official.md) and [latest documentation](http://mockito.github.io/mockito/docs/current/org/mockito/Mockito.html).
 
-### Project status
+### Project status updates
 
-* Continuously [delivering new version](https://github.com/mockito/mockito/blob/master/doc/release-notes/official.md) for every git push that passes the build
-* Moving away from [Google Code](https://code.google.com/p/mockito/) to [GitHub](https://github.com/mockito/mockito) (GitHub, you're awesome, thank you!)
-* Releasing [2.0 beta](https://github.com/mockito/mockito/issues/123), 2.0 final when ready
-* Compatible with JDK8 with exception of defender methods, JDK8 support will improve in 2.0
+Project status updates will be announced in various channels, including the [mailing list] <img alt="Google Groups" src="https://groups.google.com/forum/favicon.ico" width="21" height="21"> and [Twitter](https://twitter.com/mockitojava) <img alt="Twitter logo" src="https://g.twimg.com/dev/img/marketing/twitter-for-websites/header-logo.png" width="21" height="21">
 
 ### Contributions welcome
 
 Fancy getting world-wide visibility and building up an eternal fame of an OSS contributor?
 
 * Help the [project site](https://github.com/mockito/mockito.github.io) look decent and match the quality of the library!
 * [Contribute](https://github.com/mockito/mockito/blob/master/.github/CONTRIBUTING.md) a pull request! It is automatically released via state-of-art [continuous delivery automation](http://szczepiq.blogspot.com/2014_08_01_archive.html).
-* Use latest version! Hack and experiment. Speak up at the [mailing list](http://groups.google.com/group/mockito). Report [issues and new features](https://github.com/mockito/mockito/issues).
+* Use latest version! Hack and experiment. Speak up at the [mailing list]. Report [issues and new features](https://github.com/mockito/mockito/issues).
 
 <span id="forkongithub">
   <a href="{{ site.source_link }}" class="bg-green">
     Fork me on GitHub
   </a>
 </span>
+
+
+[mailing list]: https://groups.google.com/group/mockito
```

Comment:
Yeah looks like SO markdown, it would be nice to transform the rest of the site.